rsp/runtimes/bundles/org.jboss.tools.rsp.server.tomcat/src/main/resources/servers.json

```diff
@@ -74,6 +74,12 @@
 						"pollerProperties": {
 							"url": "http://localhost:8080"
 						}
+					},
+					"actions": {
+						"showinbrowser": {},
```

Comment:
showInBrowser action currently supports deployments, but no strategy is set for how to determine the path of a deployment. The current code here seems to remove a .ear or .war suffix and just append that to the base url. This might not be appropriate. We could have stuff like strategy=appendDeploymentName  or  strategy=appendDeploymentNameRemoveSuffix